sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/__init__.py

```diff
@@ -6,18 +6,16 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-from ._client import LogsIngestionClient
+from ._patch import LogsIngestionClient
 
-try:
-    from ._patch import __all__ as _patch_all
-    from ._patch import *  # pylint: disable=unused-wildcard-import
-except ImportError:
-    _patch_all = []
+
+from ._patch import LogsUploadError
 from ._patch import patch_sdk as _patch_sdk
 
 __all__ = [
+    "LogsUploadError",
```

Comment:
Are we adding new stuff?

Reply:
Nothing new added, and the public API should remain the same. I believe this is the result of of the autorest-postprocess step that was run to get around mypy not seeing certain methods ([context](https://github.com/Azure/autorest.python/issues/2321#issuecomment-1877921038)).